models/wiki.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var (
-	reservedWikiNames = []string{"_pages", "_new", "_edit", "raw"}
+	reservedWikiNames = []string{"_pages", "_new", "_edit", "_upload", "raw"}
```

Comment:
There should be a migration check or something if we add a new reserved name.

Reply:
maybe we should reverse all word begin with `_`. :)